src/index.html

```diff
@@ -7,6 +7,353 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home" class="header__logo-link">
+          <img 
+           src="images/logo.png"
+           alt="logo"
+           class="header-logo-img"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__menu-item">
+            <a class="is-active nav__menu-link" href="#">Apple</a>
+          </li>
+          <li class="nav__menu-item">
+            <a class="nav__menu-link" href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__menu">
          <li class="nav__item">
            <a class="is-active nav__link" href="#">Apple</a>
          </li>
          <li class="nav__item">
            <a class="nav__link" href="#">Samsung</a>
          </li>
```
why so complicated?
no need to complicate BEM structure
fix it please in whole file